spec/eof.md

```diff
@@ -214,11 +214,13 @@ The following instructions are introduced in EOF code:
 - `EOFCREATE (0xec)` instruction
```

Comment:
This also applies to `TXCREATE`, so it should be copied there.

Reply:
TXCREATE is defined by building on top of EOFCREATE here, so these changes apply to TXCREATE as well.

Reply:
Ok, the EIP has a different layout, which is what I had on my mind.

Reply:
Yea, I think this is not the first time this layout throws people off, we might want to migrate to the EIP's one at some future point.